astropy/tests/helper.py

```diff
@@ -191,7 +179,7 @@ def enable_deprecations_as_exceptions(include_astropy_deprecations=True,
 
     warnings_to_ignore_by_pyver : dict
         Dictionary mapping tuple of ``(major, minor)`` Python version to
-        a list of deprecation warning messages to ignore.
+        a list of ``(warning_message, warning_class)`` to ignore.
```

Comment:
Note: This is an API change without deprecation... 😬

Reply:
We'll see whether it bites anyone downstream. Won't affect users but maintainers, and they know where to complain about it, so I'm not much worried.